Sources/Prometheus/MetricTypes/Histogram.swift

```diff
@@ -144,7 +213,21 @@ public class PromHistogram<NumType: DoubleRepresentable, Labels: HistogramLabels
             }
         }
     }
-
+    
+    /// Time the duration of a closure and observe the resulting time in seconds.
+    ///
+    /// - parameters:
+    ///     - labels: Labels to attach to the resulting value.
+    ///     - body: Closure to run & record.
+    @inlinable
+    public func time<T>(_ labels: Labels? = nil, _ body: @escaping () throws -> T) rethrows -> T {
```

Comment:
Not sure we really need this but if you want to include it then ok

Reply:
Not my choice, requirement set forth by Prometheus 😄

Reply:
recommended I guess, but yeah now I noticed it's a SHOULD have the methods -- good to add then! Thanks!